Cargo.toml

```diff
@@ -1,6 +1,5 @@
 [workspace]
 members = [
   "src/services/fluence-aurora-connector/modules/curl_adapter",
-  "src/services/fluence-aurora-connector/modules/fluence_aurora_connector",
-  "src/distro"
```

Comment:
Why? Is it working without it?

Reply:
o_O I've no idea.

Reply:
I suppose CLion removed that when I was renaming some files. I have no other ideas 🤔